extensions/default/src/Panels/Toolbox/ToolboxUI.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { PanelSection, ToolSettings, Tooltip } from '@ohif/ui';
+import classnames from 'classnames';
+
+function ToolboxUI(props) {
```

Comment:
Needs docs as to how this gets used, as this is supposed to be a generic component.